### PR TITLE
Fixed path in logo customization section

### PR DESCRIPTION
**Pull request by chantal-kelm**
A new constant has been created in the `common/constants.ts` file and used in all necessary places

### CODE REVIEW

**Review by AlexRuiz7 — APPROVED** [review]
LGTM!

**Comment by matiasmoreno876** [comment]
In this PR it is necessary to backport to the 4.3-7.16 branches and 4.3-1.2-wdz?

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

**Review by AlexRuiz7 — APPROVED** [review]
- Code review: ✔️ 
- Test: ✔️

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.3.7 - Kibana 7.10.2, 7.16.x, 7.17.x - Revision 4308
+
+### Fixed
+- Path fix in Logo Customization section [#4352](https://github.com/wazuh/wazuh-kibana-app/pull/4352)
```
Comment (gdiazlo) [inline CHANGELOG.md:8]: Please update the changelog due to PR rename.

Comment (AlexRuiz7) [reply CHANGELOG.md]: Done. Thanks!

**Review by Desvelao — APPROVED** [review]
**review**
Code :heavy_check_mark: 
Test :heavy_check_mark: 

**Screenshots**
image

```
# ------------------------------ wazuh-logo-customization -------------------------------
#
#Define the name of the app logo saved in the path /plugins/wazuh/public/assets/
#customization.logo.app: ''
#
#Define the name of the sidebar logo saved in the path /plugins/wazuh/public/assets/
#customization.logo.sidebar: ''
#
#Define the name of the health-check logo saved in the path /plugins/wazuh/public/assets/
#customization.logo.healthcheck: ''
#
#Define the name of the reports logo (.png) saved in the path /plugins/wazuh/public/assets/
#customization.logo.reports: ''
```

**Comment by Desvelao** [comment]
> The backport to `4.3-1.2-wzd` failed:
> 
> ```
> The process '/usr/bin/git' failed with exit code 1
> ```
> 
> To backport manually, run these commands in your terminal:
> 
> ```shell
> # Fetch latest updates from GitHub
> git fetch
> # Create a new working tree
> git worktree add .worktrees/backport-4.3-1.2-wzd 4.3-1.2-wzd
> # Navigate to the new working tree
> cd .worktrees/backport-4.3-1.2-wzd
> # Create a new branch
> git switch --create backport-4352-to-4.3-1.2-wzd
> # Cherry-pick the merged commit of this pull request and resolve the conflicts
> git cherry-pick -x --mainline 1 05598cf83a70221e8203b2b41afd43d61813fee6
> # Push it to GitHub
> git push --set-upstream origin backport-4352-to-4.3-1.2-wzd
> # Go back to the original working tree
> cd ../..
> # Delete the working tree
> git worktree remove .worktrees/backport-4.3-1.2-wzd
> ```
> 
> Then, create a pull request where the `base` branch is `4.3-1.2-wzd` and the `compare`/`head` branch is `backport-4352-to-4.3-1.2-wzd`.

Integrated in 4f806e367338c37a3c01c7bc08f64c9f5b19804d.

**Pull request merged**